src/subsplit_dag.cpp

```diff
@@ -488,13 +462,20 @@ void SubsplitDAG::BuildNodes(const SizeBitsetMap &index_to_child,
   for (const auto &rootsplit : rootsplits) {
     BuildNodesDepthFirst(index_to_child, rootsplit, visited_subsplits);
   }
+
+  // Finally, we add the DAG root node. The DAG root node's subsplit bitset
```

Comment:
This is a great comment, but could we put it somewhere a little more visible? Also, I think that I suggested above that we just factor this bitset creation into a Bitset function?